### PR TITLE
Feature/mijn 1607 append data

### PR DESCRIPTION
**Pull request by drakonen**
Does not include adresHistorisch because i do not have data for this.

### CODE REVIEW

File: mks/model/stuff.py [hunk 1]
```diff
@@ -243,12 +258,8 @@ def get_partner(self):
             {'name': 'geslachtsaanduiding', 'parser': self.to_string},
             {'name': 'geboortedatum', 'parser': self.to_date},
             {'name': 'overlijdensdatum', 'parser': self.to_date},
-        ]
+            {'name': 'adellijkeTitelPredikaat', 'parser': self.to_date},
```
Comment (timvanoostrom) [inline mks/model/stuff.py]: Shouldn't this be `to_string`?

File: mks/model/stuff.py [hunk 2]
```diff
@@ -289,23 +313,60 @@ def get_kinderen(self):
             {'name': 'geslachtsnaam', 'parser': self.to_string},
             {'name': 'geslachtsaanduiding', 'parser': self.to_string},
             {'name': 'geboortedatum', 'parser': self.to_date},
+            {'name': 'inp.geboorteplaats', 'parser': self.to_date, 'save_as': 'geboorteplaats'},
+            {'name': 'inp.geboorteLand', 'parser': self.to_date, 'save_as': 'geboorteLand'},
             {'name': 'overlijdensdatum', 'parser': self.to_date},
+            {'name': 'adellijkeTitelPredikaat', 'parser': self.to_date},
```
Comment (timvanoostrom) [inline mks/model/stuff.py]: Same here, shouldn't this be `self.to_string` ?

File: mks/model/stuff.py [hunk 3]
```diff
@@ -289,23 +313,60 @@ def get_kinderen(self):
             {'name': 'geslachtsnaam', 'parser': self.to_string},
             {'name': 'geslachtsaanduiding', 'parser': self.to_string},
             {'name': 'geboortedatum', 'parser': self.to_date},
+            {'name': 'inp.geboorteplaats', 'parser': self.to_date, 'save_as': 'geboorteplaats'},
```
Comment (timvanoostrom) [inline mks/model/stuff.py]: Also here `self.to_date` seems incorrect for a non date value

File: mks/model/stuff.py [hunk 4]
```diff
@@ -289,23 +313,60 @@ def get_kinderen(self):
...
+
+            result.append(ouder)
+
+        result.sort(key=lambda x: x['geboortedatum'])
+
+        return result
+
     def get_nationaliteiten(self):
         if not self.nationaliteiten:
             return {}
```
Comment (timvanoostrom) [inline mks/model/stuff.py]: Will this be `jsonified` to a `[]`?

File: mks/swagger.yaml [hunk 5]
```diff
@@ -150,20 +154,59 @@ definitions:
     title: kind
     type: object
     properties:
+      adellijkeTitelPerdikaat:
```
Comment (timvanoostrom) [inline mks/swagger.yaml]: typo adellijkeTitelP`re`dikaat

File: mks/model/stuff.py [hunk 6]
```diff
@@ -98,10 +98,10 @@ def _process_response(self, resp: ElementTree) -> None:
                 self._base_paths['partner'] +
                 self._base_paths['gerelateerde'])
 
-            # self.gerelateerde_ouder = objectify.ObjectPath(
-            #     self._base_paths['ouders'] +
-            #     self._base_paths['gerelateerde'])
-            #
+            self.gerelateerde_ouder = objectify.ObjectPath(
+                self._base_paths['ouders'] +
+                self._base_paths['gerelateerde'])
```
Comment (timvanoostrom) [inline mks/model/stuff.py]: What data does `gerelateerde` include ?

File: mks/model/stuff.py (hunk 1 above)
Comment (drakonen) [reply mks/model/stuff.py]: copy pasta mistake :hear_no_evil:

File: mks/model/stuff.py (hunk 2 above)
Comment (drakonen) [reply mks/model/stuff.py]: Copy pasta mistake :see_no_evil:

File: mks/model/stuff.py (hunk 3 above)
Comment (drakonen) [reply mks/model/stuff.py]: copy pasta mistake :speak_no_evil:

File: mks/model/stuff.py (hunk 4 above)
Comment (drakonen) [reply mks/model/stuff.py]: should be a `[]` indeed

File: mks/swagger.yaml (hunk 5 above)
Comment (drakonen) [reply mks/swagger.yaml]: fixed

File: mks/model/stuff.py (hunk 6 above)
Comment (drakonen) [reply mks/model/stuff.py]: `self._gerelateerde_ouder` is internal, it references the 'gerelateerde' xml stuff in the stuf data.

**Pull request merged**